snowpack/src/scan-imports.ts

```diff
@@ -132,17 +132,17 @@ function cleanCodeForParsing(code: string): string {
 }
 
 function parseJsForInstallTargets(contents: string): InstallTarget[] {
-  let imports: ImportSpecifier[];
+  let imports: ImportSpecifier[] = [];
   // Attempt #1: Parse the file as JavaScript. JSX and some decorator
   // syntax will break this.
   try {
-    [imports] = parse(contents) || [];
```

Comment:
In upgrading TypeScript for the templates, upgrading TypeScript for the core repo exposed an existing bug. This doesn’t change any code; just fixed a now-visible type bug (fortunately there was no runtime bug, but I’m glad it caught this nonetheless)